register/register.go

```diff
@@ -0,0 +1,25 @@
+package register
+
+import (
+	"context"
+
+	"github.com/slsa-framework/slsa-verifier/options"
+)
+
+var SLSAVerifiers = make(map[string]SLSAVerifier)
+
+type SLSAVerifier interface {
+	// Match matches a BuilderID.
+	Match(builderID string) bool
+
+	// Verify verifies a provenance.
+	Verify(ctx context.Context,
```

Comment:
Should we qualify this with `VerifyArtifact` and comment `Verify verifies a provenance for a supplied artifact` 

(vs. `VerifyImage` that verifies a container image with provenance stored in the registry?)

Reply:
sure. I left it as-is because we can change in your upcoming PR. But I'll do it here anyway. Thanks